13/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -352,7 +352,6 @@ run_pgupgrade ()
   info_msg "Starting old postgresql once again for a clean shutdown..."
   "${old_pgengine}/pg_ctl" start -w --timeout 86400 -o "-h ''"
   info_msg "Waiting for postgresql to be ready for shutdown again..."
-  "${old_pgengine}/pg_isready"
```

Comment:
Why are you removing this safety procedure?

Reply:
On my testing it returned exit code 2 - I did manual checks the socket was up and running but the `pg_isready` wasn't catching it.
On the above line we use `-w` which is waiting for the DB to be ready. I thought maybe we don't need to double check it unless there is another reason for it?

Reply:
after some searching, It seems we start the server as local only -- which mean it doesn't create a TCP port instead it creates a Linux socket.

the 'pg_isready' tries the port and fails

Reply:
fixed

Reply:
+1